src/components/TestResult/ResultBody.tsx

```diff
@@ -22,29 +22,31 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import { EuiFlexItem, EuiText } from "@elastic/eui";
+import { EuiFlexGroup, EuiFlexItem, EuiText } from "@elastic/eui";
 import React from "react";
 import type { StepStatus } from "../../common/types";
-import { ResultContentWithoutAccordion, symbols } from "./styles";
+import { ResultContentWrapper, symbols } from "./styles";
 
 interface IResultBody {
-  durationElement: JSX.Element;
-  name: string;
+  actionTitles: string[];
   resultCategory: StepStatus;
 }
 
-export function ResultBody({
-  durationElement,
-  name,
-  resultCategory,
-}: IResultBody) {
+export function ResultBody({ actionTitles, resultCategory }: IResultBody) {
   return (
-    <ResultContentWithoutAccordion alignItems="center" gutterSize="xs">
-      <EuiFlexItem grow={false}>{symbols[resultCategory]}</EuiFlexItem>
-      <EuiFlexItem>
-        <EuiText size="s">{name}</EuiText>
-      </EuiFlexItem>
-      <EuiFlexItem grow={false}>{durationElement}</EuiFlexItem>
-    </ResultContentWithoutAccordion>
+    <EuiFlexGroup direction="column" gutterSize="none">
+      {actionTitles.map((name, index) => (
+        <ResultContentWrapper
+          alignItems="center"
+          key={name + index}
+          gutterSize="xs"
+        >
+          <EuiFlexItem grow={false}>{symbols[resultCategory]}</EuiFlexItem>
+          <EuiFlexItem>
+            <EuiText size="s">{name}</EuiText>
+          </EuiFlexItem>
+        </ResultContentWrapper>
+      ))}
+    </EuiFlexGroup>
```

Comment:
Isn't this the same as the first `EuiFlexGroup` in `src/components/TestResult/ResultErrorBody.tsx`?

Shouldn't we reuse this down there?

Reply:
Yes, it's the same exact props. In my opinion, it's not worth reusing because there's nothing novel here in the way the props are defined and there's no custom styling or other modification of the underlying functionality.

I think if the flex group's props were computed or if we had added some custom style it could be worthwhile to extract it and reuse it.

I don't think there's any major compute savings either, is there an aspect to this I'm overlooking?

Reply:
No major compute savings indeed. This came to mind just to keep code "DRY" given it's the same markup and arguments. Given it's an extra indirection, however, I'm not strongly opinionated about making it a separate component unless we start using it more often.

Reply:
I'm usually alright with having a single repetition elsewhere in the codebase. If we were to start doing this more than that, then I think it's worth maintaining a separate file for it. Just my opinion though.